src/app/technical-record/tech-rec-history/tech-rec-history.component.spec.ts

```diff
@@ -1,36 +1,35 @@
-import { ComponentFixture, getTestBed, TestBed } from '@angular/core/testing';
-import { SharedModule } from '@app/shared/shared.module';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { TechRecHistoryComponent } from './tech-rec-history.component';
 import { VehicleTechRecordModel } from '@app/models/vehicle-tech-record.model';
 import { TESTING_UTILS } from '../../utils/testing.utils';
+import { OrderByStatusPipe } from '@app/pipes/OrderByStatusPipe';
+import { DefaultNullOrEmpty } from '@app/pipes/DefaultNullOrEmptyPipe';
+
+const mockedTechRecord = TESTING_UTILS.mockTechRecord({
+  statusCode: 'current',
+  reasonForCreation: 'reason for creation',
+  createdByName: 'test name',
+  createdAt: '1-1-2020',
```

Comment:
your mock dates should be proper format otherwise your generated snaptshot will not have the correct display and we wont whether we doing the correct thing

Reply:
you need to check: YYYY-MM-DD